.circleci/config.yml

```diff
@@ -17,6 +17,26 @@ jobs:
       - checkout
       - run: curl -L https://git.io/vp6lP | sh # gometalinter
       - run: PATH=./bin:$PATH gometalinter --deadline=5m --disable-all --enable=gofmt --enable=vet --vendor ./...
+  # FIXME: this is pinned to v3.0.0 because subsequent versions of ct pull in new versions of helm, which are subject
+  #        to this bug: https://github.com/helm/helm/issues/8596, specifically for rbac object names with ':' in them
```

Comment:
This appears to be fixed 
https://github.com/helm/helm/issues/8596

Reply:
Still appears to be a problem: https://github.com/helm/helm/issues/8835

Reply:
Ahh okay, can we update the comment?

Reply:
Yeah, I'll do that.